test/integration/test_bind.rb

```diff
@@ -1,12 +1,16 @@
 require_relative '../test_helper'
 
 class TestBindIntegration < LDAPIntegrationTestCase
+
+  INTEGRATION_HOSTNAME = 'ldap.example.org'.freeze
+
   def test_bind_success
     assert @ldap.bind(BIND_CREDS),
            @ldap.get_operation_result.inspect
   end
 
   def test_bind_timeout
+    omit "this is no longer working in our test environment - skipping"
```

Comment:
:(

Reply:
Yeah 😢 I'm getting a `connection refused`, which makes total sense, because the test is checking on a closed port. I didn't find any evidence the script set up anything in that port, so begs the question _why did it work this way in first place_ 🤔 🤷‍♂

Reply:
finally solved it using a non-routable IP! 🎉